sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/patch/PatchUtil.java

```diff
@@ -57,6 +58,16 @@ private static JsonSerializable cosmosPatchToJsonSerializable(CosmosPatchOperati
 
         jsonSerializable.set("operations", operations);
 
+        if(cosmosPatchSpec.getRequestOptions() != null)
+        {
```

Comment:
code style: '{' should be on the same line as if/while/try/catch/finally/method definition